src/pudl/package_data/ferc1/xbrl_factoid_plant_function_tags.csv

```diff
@@ -1,29 +1,35 @@
 table_name,xbrl_factoid,aggregatable_plant_function
-plant_in_service_ferc1,intangible_plant,intangible
-plant_in_service_ferc1,steam_production_plant,steam_production
-plant_in_service_ferc1,nuclear_production_plant,nuclear_production
-plant_in_service_ferc1,hydraulic_production_plant,hydraulic_production
-plant_in_service_ferc1,other_production_plant,other_production
-plant_in_service_ferc1,transmission_plant,transmission
-plant_in_service_ferc1,distribution_plant,distribution
-plant_in_service_ferc1,transmission_and_market_operation_plant_regional_transmission_and_market_operation_plant,regional_transmission_and_market_operation
-plant_in_service_ferc1,general_plant,general
-electric_operating_expenses_ferc1,steam_power_generation_operations_expense,steam_production
-electric_operating_expenses_ferc1,steam_power_generation_maintenance_expense,steam_production
-electric_operating_expenses_ferc1,nuclear_power_generation_operations_expense,nuclear_production
-electric_operating_expenses_ferc1,nuclear_power_generation_maintenance_expense,nuclear_production
-electric_operating_expenses_ferc1,hydraulic_power_generation_operations_expense,hydraulic_production
-electric_operating_expenses_ferc1,hydraulic_power_generation_maintenance_expense,hydraulic_production
-electric_operating_expenses_ferc1,other_power_generation_operations_expense,other_production
-electric_operating_expenses_ferc1,other_power_generation_maintenance_expense,other_production
-electric_operating_expenses_ferc1,other_power_supply_expense,
-electric_operating_expenses_ferc1,transmission_operation_expense,transmission
-electric_operating_expenses_ferc1,transmission_maintenance_expense_electric,transmission
-electric_operating_expenses_ferc1,regional_market_operation_expense,regional_transmission_and_market_operation
-electric_operating_expenses_ferc1,regional_market_maintenance_expense,regional_transmission_and_market_operation
-electric_operating_expenses_ferc1,distribution_operation_expenses_electric,distribution
-electric_operating_expenses_ferc1,distribution_maintenance_expense_electric,distribution
-electric_operating_expenses_ferc1,customer_account_expenses,
-electric_operating_expenses_ferc1,customer_service_and_information_expenses,
-electric_operating_expenses_ferc1,sales_expenses,
-electric_operating_expenses_ferc1,administrative_and_general_expenses,general
+core_ferc1__yearly_plant_in_service_sched204,intangible_plant,intangible
```

Comment:
Since we're using the table names with the schedules here (which differ from the table names in PUDL), is there a translation step I'm not seeing here?

Reply:
we ended up landing on table names with the schedules in them!

Reply:
its a bit of a divergence from the other tables but it ended up feeling like the easiest way to communicate to users which DBF or XBRL table the data originated from.

Reply:
ah then I'm hopelessly out of date post rename!